dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -95,46 +85,55 @@ protected void addFilesToWorkflowAttachment(List<File> workflowAttachment, File
             exceptionMessage(e, "Unable to get workflow attachment files from zip file " + zippedEntry.getName(), IO_ERROR);
         }
 
-        // Put file names in workflow attachment list
-        File[] listOfFiles = tempDir.listFiles();
-        for (int i = 0; i < listOfFiles.length; i++) {
-            if (listOfFiles[i].isFile()) {
-                // TODO check file extension (type) only for a local entry; pass all files found in remote entry folder?
-                // Locate code that grabs all imports for a non local entry and use that instead of checking extension
-                // since CWL can import many file types
-                // There may be confidential or large files that are not needed in a local directory that should
-                // not be sent to a remote endpoint?
-                if (fileIsCorrectType(listOfFiles[i])) {
-                    System.out.println("Adding file " + listOfFiles[i].getName() + " to workflow attachment");
-                    File fileToAdd = new File(tempDir, listOfFiles[i].getName());
-                    workflowAttachment.add(fileToAdd);
-                } else {
-                    System.out.println("File " + listOfFiles[i].getName() + " is not the correct type for the workflow so it will not be "
-                            + "added to the workflow attachment");
-                }
-            } else if (listOfFiles[i].isDirectory()) {
-                System.out.println("Found directory " + listOfFiles[i].getName());
+        try {
+
+            String[] fileFilter = null;
```

Comment:
Can you please add a comment saying that a null fileFilter causes all files to be returned? When I first looked at this, I thought it would result in an NPE, and had to go look at the code to verify it wouldn't.